eof/dataspace_client.py

```diff
@@ -224,6 +227,7 @@ def _construct_orbit_file_query(
     query_template = (
         "startswith(Name,'{mission_id}') and contains(Name,'{orbit_type}') "
         "and ContentDate/Start lt '{start_time}' and ContentDate/End gt '{stop_time}'"
+        # " and productType eq {orbit_type}"
```

Comment:
Is this redundant because the current query template has `contains(Name,'{orbit_type}')`? or was this a mistake to comment it out?

Reply:
IIRC, this was something I wanted to test but I didn't find/take the time to do it.  So far you were using `contains(Name,'{orbit_type}')` and while reading the doc, I wondered whether `productType eq {orbit_type}` would have been enough.

I shouldn't have commited it, but forgot about it.